### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by ggabernet**
Many thanks to contributing to qbicsoftware/rnadeseq!

Please fill in the appropriate checklist below (delete whatever is not relevant). These are the most common things requested on pull requests (PRs).

## PR checklist
 - [ ] This comment contains a description of changes (with reason)
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
 - [ ] Documentation in `docs` is updated
 - [ ] `CHANGELOG.md` is updated
 - [ ] `README.md` is updated

**Learn more about contributing:** https://github.com/qbicsoftware/rnadeseq/tree/master/.github/CONTRIBUTING.md

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -32,18 +30,18 @@ There are typically two types of tests that run:
...
+The qbic-pipelines has a [set of guidelines](http://nf-co.re/guidelines) which all pipelines must adhere to.
+To enforce these and ensure that all pipelines stay in sync, we have developed a helper tool which runs checks on the pipeline code. This is in the [qbic-pipelines/tools repository](https://github.com/qbic-pipelines/tools) and once installed can be run locally with the `qbic-pipelines lint <pipeline-directory>` command.
 
 If any failures or warnings are encountered, please follow the listed URL for more documentation.
 
 ### Pipeline Tests
 
-Each qbicsoftware pipeline should be set up with a minimal set of test-data.
+Each qbic-pipelines pipeline should be set up with a minimal set of test-data.
 Travis CI then runs the pipeline on this data to ensure that it exists successfully.
```
Comment (FriederikeHanssen) [inline .github/CONTRIBUTING.md:41]: Are you not setting up GHA for this?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -1,60 +1,86 @@
...
+      - name: Check if Dockerfile or Conda environment changed
+        uses: technote-space/get-diff-action@v1
+        with:
+          PREFIX_FILTER: |
+            Dockerfile
+            environment.yml
+
+      - name: Build new docker image
+        if: env.GIT_DIFF
+        run: docker build --no-cache . -t nfcore/bcellmagic:dev
```
Comment (FriederikeHanssen) [inline .github/workflows/ci.yml:34]:
```suggestion
        run: docker build --no-cache . -t qbicpipelines/rnadeseq:dev
```

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -1,60 +1,86 @@
...
+      - name: Check if Dockerfile or Conda environment changed
+        uses: technote-space/get-diff-action@v1
+        with:
+          PREFIX_FILTER: |
+            Dockerfile
+            enfironment.yml
+      
+      - name: Build new docker image
+        if: env.GIT_DIFF
+        run: docker build --no-cache . -t nfcore/bcellmagic:dev
```
Comment (FriederikeHanssen) [inline .github/workflows/ci.yml:72]:
```suggestion
        run: docker build --no-cache . -t qbicpipelines/rnadeseq:dev
```

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -1,60 +1,86 @@
...
-        run: nextflow run ${GITHUB_WORKSPACE} -profile test_contrast_matrix,docker
-  contrast_list:
-    runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        nxf_ver: ['19.10.0']
-    steps:
-      - uses: actions/checkout@v1
+          docker pull qbicpipelines/rnadeseq:dev
+          docker tag qbicpipelines/rnadeseq:dev qbicpipelines/rnadeseq:dev
```
Comment (FriederikeHanssen) [inline .github/workflows/ci.yml:78]: Since this is a PR to master, doesn't this need to have a release number tag?

File: CODE_OF_CONDUCT.md [hunk 5]
```diff
@@ -34,7 +34,7 @@ This Code of Conduct applies both within project spaces and in public spaces whe
 
 ## Enforcement
 
-Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team on [Slack](https://nf-core-invite.herokuapp.com/). The project team will review and investigate all complaints, and will respond in a way that it deems appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team on [Slack](https://qbic-pipelines-invite.herokuapp.com/). The project team will review and investigate all complaints, and will respond in a way that it deems appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
```
Comment (FriederikeHanssen) [inline CODE_OF_CONDUCT.md:37]: cool, do we an extra qbic-pipelines slack now? :) If so, I should add it to my pipeline too

File: README.md [hunk 6]
```diff
@@ -1,20 +1,18 @@
-# qbicsoftware/rnadeseq
+# qbic-pipelines/rnadeseq
```
Comment (FriederikeHanssen) [inline README.md:1]: If you like to, could also add the title image here as in bamtofastq or remove it there, to have a similar look. However, I don't think it is as important to have for our organisation

**Review by FriederikeHanssen — COMMENTED** [review]
Nice! I think you found all the qbicsoftware spots. 

I won't approve just yet, because I am pretty sure that the wrong container is pulled (bcellmagic), which will not go well. 
There is no branch protection GHA. Is this intentional?

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (ggabernet) [reply .github/CONTRIBUTING.md]: yes, good catch!

File: CODE_OF_CONDUCT.md (hunk 5 above)
Comment (ggabernet) [reply CODE_OF_CONDUCT.md:37]: No we don't , I was just batch updating nf-core to qbic-pipelines 😄

File: README.md (hunk 6 above)
Comment (ggabernet) [reply README.md:1]: yeah why not, it looks really cool in your pipeline 😄

**Review by FriederikeHanssen — APPROVED** [review]

**Pull request merged**